gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationFragment.java

```diff
@@ -94,9 +94,11 @@ public void onViewCreated(
     ((MainActivity) getActivity()).setActionBar(toolbar, R.drawable.ic_close_black_24dp);
     toolbar.setNavigationOnClickListener(__ -> onCloseButtonClick());
     // Observe state changes.
-    viewModel.getForm().observe(this, this::rebuildForm);
-    viewModel.getToolbarTitle().observe(this, toolbar::setTitle);
-    viewModel.getSaveResults().observe(this, e -> e.ifUnhandled(this::handleSaveResult));
+    viewModel.getForm().observe(getViewLifecycleOwner(), this::rebuildForm);
```

Comment:
Why is this necessary? Isn't the Fragment the LifecycleOwner?

Reply:
This is needed because of the updated lifecycle library: 2.1.0 --> 2.2.0

Reply:
Let's revert this and handle it separately

Reply:
I'm fine slotting it in, was just wondering why it was needed.

Reply:
FYI, that wasn't breaking the assemble build, but the lint check.
https://stackoverflow.com/a/58912855

Reply:
I see, this is very helpful, thank you!